packages/aws-cdk-lib/aws-ecs/lib/amis.ts

```diff
@@ -395,13 +422,14 @@ export class BottleRocketImage implements ec2.IMachineImage {
     this.amiParameterName = `/aws/service/bottlerocket/${this.variant}/${this.architecture}/latest/image_id`;
 
     this.cachedInContext = props.cachedInContext ?? false;
+    this.additionalCacheKey = props.additionalCacheKey;
```

Comment:
We should probably add some validation to say that `additionalCacheKey` only makes sense if `cachedInContext==true` ?